source/draw/gpu/opengl/OpenGLWindow.ooc

```diff
@@ -13,7 +13,24 @@ use opengl
 use base
 
 version(!gpuOff) {
-OpenGLWindow: class extends OpenGLSurface {
+OpenGLWindow: class extends GpuCanvas {
+	_toLocal := FloatTransform3D createScaling(1.0f, -1.0f, -1.0f)
+	context ::= this _context as OpenGLContext
+	drawLines: override func ~explicit (pointList: VectorList<FloatPoint2D>, pen: Pen) { raise("Cannot draw lines directly to a window.") }
+	drawPoints: override func ~explicit (pointList: VectorList<FloatPoint2D>, pen: Pen) { raise("Cannot draw lines directly to a window.") }
```

Comment:
Line and point drawing is not needed directly to the window since the interface just draws an image to the window when done.